packages/vertix-base/src/bases/object-base.ts

```diff
@@ -0,0 +1,51 @@
+import { ForceMethodBase } from "@vertix.gg/base/src/errors/force-method-implementation";
```

Comment:
/ask create an issue that is todo to move this file to another repo called @zenflux/core